### ISSUE TITLE
Issue: The power button doesn't switch on the device

### ISSUE THREAD

**Issue by prutapptic** [openstf/openstf.github.io#4]
Hello,
I would like to repport a bug i'm facing with the Sony Xperia S on Android 4.0.4:
I control the Smartphone without any problem. I lock it with the "Power" button of the "Advanced" tab. 
Now it's impossible to control it. I would like to unlock the device with this same "Power" button but a clic on it has no effect. I didn't find any way to unlock a device with STF.

**Comment by sorccu** [comment]
Yeah, usually when you press the power button it stops listening for virtual button presses entirely (including the power button). There's a way to wake up the device but it's not exposed in the UI right now.

By the way this is the repo for the org site, can you file bugs against https://github.com/openstf/stf?
